_po/ja/how-to/laravel.po

```diff
@@ -878,6 +1330,20 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"`lang/ja.json`\n"
+"```json\n"
+"{\n"
+"    \"Title\": \"タイトル\",\n"
+"    \"Body\": \"本文\",\n"
+"    \"Search\": \"検索\",\n"
+"    \"Enter search keyword\": \"検索キーワードを入力\",\n"
+"    \"of\": \"件目、全：\",\n"
+"    \"per page\": \"毎ページ\",\n"
+"    \"results\": \"件\",\n"
+"    \"to\": \"から\",\n"
+"    \"Showing\": \"表示中：\"\n"
+"}\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.